Headlines/src/Controllers/NewsTableViewController.swift

```diff
@@ -574,9 +594,4 @@ class NewsTableViewController: UITableViewController,
         vc.delegate = self
         present(vc, animated: true, completion: nil)
     }
-    
-    // MARK: - TabbedViewController
-    func tabbedViewControllerWasDoubleTapped() {
-        tableView.setContentOffset(CGPoint.zero, animated: true)
-    }
 }
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/Canillitapp/headlines-iOS/blob/a319000992780f90ea2d6523355ead4415bf6702/Headlines/src/Controllers/NewsTableViewController.swift#L597"></span>File should contain 400 lines or less: currently contains 597</td>
    </tr>
  </tbody>
</table>